plugins/httpapi/mso.py

```diff
@@ -11,16 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+#
```

Comment:
Remove the Apache License as it does not apply to this file at it is in a GPL repo.